polyfill/lib/calendar.mjs

```diff
@@ -117,7 +117,11 @@ export class Calendar {
     return GetSlot(this, CALENDAR_ID);
   }
   static from(item) {
-    if (ES.Type(item) === 'Object') return item;
+    if (ES.Type(item) === 'Object') {
+      if (!('calendar' in item)) return item;
```

Comment:
Prefer getting the property once and checking against `undefined`.

Reply:
I think that considering an object with `calendar: undefined` to signify a custom calendar would work, but it is different from the semantic that we landed on, where it's the shape of the object that matters: if a `calendar` property is present then it's a property bag, if absent then it's a custom calendar.